src/distilabel/pipeline/local.py

```diff
@@ -486,6 +490,8 @@ def _request_initial_batches(self) -> None:
 
         for step in self._batch_manager._steps.values():
             if batch := step.get_batch():
+                if self._dry_run:
+                    batch.last_batch = True
```

Comment:
At the end we don't need this, because we're passing the `_dry_run` to the `_ProcessWrapper`